### PR TITLE
Add InsightFinder Integration Tile

### PR DESCRIPTION
**Pull request by dearmore**
### What does this PR do?

Adds InsightFinder integration tile

### CODE REVIEW

File: insightfinder/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+For support, contact us at [support@insightfinder.com][4].
+
+## Further Reading
+
+Additional helpful documentation, links, and articles:
+
+[1]: https://insightfinder.com/
+[2]: https://insightfinder.com/datadog-integration/
+[3]: https://app.insightfinder.com/auth/signup
+[4]: mailto:support@insightfinder.com
```
Comment (cswatt) [inline insightfinder/README.md:28]:
```suggestion
To configure the integration and send data to InsightFinder, see the [InsightFinder-Datadog Integration][2]. You need a Datadog [API key and application key][5].

If you do not have an InsightFinder account, [sign up][3] to start your free trial.

## Support

Contact [Datadog support][6] or email [InsightFinder support][4].

## Further Reading

Additional helpful documentation, links, and articles:

[1]: https://insightfinder.com/
[2]: https://insightfinder.com/datadog-integration/
[3]: https://app.insightfinder.com/auth/signup
[4]: mailto:support@insightfinder.com
[5]: https://docs.datadoghq.com/account_management/api-app-keys/
[6]: https://docs.datadoghq.com/help/
```

**Review by cswatt — CHANGES_REQUESTED** [review]
left some styleguide feedback!

File: insightfinder/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+If you do not have an InsightFinder account, [sign up][3] to start your free trial.
+
+## Support
+
+Contact [Datadog support][6] or email [InsightFinder support][4].
+
+## Further Reading
+
+Additional helpful documentation, links, and articles:
```
Comment (cswatt) [inline insightfinder/README.md:23]:
```suggestion
```

missed this the first time around! there aren't any further reading links on this page, so let's get rid of this section.

**Review by cswatt — APPROVED** [review]

**Comment by anawishnoff** [comment]
Hey @dearmore, thanks for putting this PR together! I've reviewed the content and it seems like the integration is only sending data into Datadog. Integrations need to be bi-directional to receive Marketplace support (which if I remember correctly was going to be the next step for your team). Would it be possible for the integration to send events or metrics into Datadog, as well as pulling data out? We can chat more about some options over email if needed. Thanks!

cc @alexvetras

**Comment by alexvetras** [comment]
Thanks, @anawishnoff. I connected with InsightFinder, and we are okay to move forward with this initial integration to get the tile live. 

InsightFinder plans to send data back into Datadog in an upcoming version, which will be covered in a separate, future pull-request.

File: insightfinder/manifest.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "guid": "1ecaeadf-7b34-48f7-b9cb-81b8a9e4e4dc",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "InsightFinder",
+  "categories": [
+    ""
```
Comment (sarah-witt) [inline insightfinder/manifest.json:19]: Do you want to add any categories for this integration?

**Review by sarah-witt — COMMENTED** [review]
Looks good, I just had one question!

File: insightfinder/manifest.json (hunk 3 above)
Comment (alexvetras) [reply insightfinder/manifest.json:19]: @dearmore, if you would like to include a category or two, please select from the options in the following word cloud: https://docs.datadoghq.com/integrations/.

Comment (dearmore) [reply insightfinder/manifest.json]: @alexvetras @sarah-witt Thanks for the feedback, I have updated the categories. Let me know if there's anything else needed. Thanks!

**Review by sarah-witt — COMMENTED** [review]
Thanks @dearmore! You can also remove the empty dashboard file `insightfinder/assets/dashboards/insightfinder_overview.json` since it will not be used.

File: insightfinder/manifest.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "windows"
+  ],
+  "public_title": "InsightFinder",
+  "categories": [
+    "automation",
+    "monitoring",
+    "notification"
+  ],
+  "type": "check",
+  "is_public": false,
```
Comment (sarah-witt) [inline insightfinder/manifest.json:24]:
```suggestion
  "is_public": true,
```
Would you like the integration to be publicly available on our documentation after merging this PR? If so, you can do so by changing this field.

**Review by sarah-witt — COMMENTED** [review]
Hi @dearmore, everything looks good! I had one last question before merging and applying. Thanks!

**Review by sarah-witt — APPROVED** [review]
Looks good!

**Pull request merged**